packages/ui/src/atom/label-atom/label-atom.vue

```diff
@@ -4,6 +4,7 @@
     :class="themeCssClass"
   >
     {{ label }}
+    <slot />
```

Comment:
If you want to pass icon - create new component `icon-label` and use additional named slot or property for icon rendering.

Reply:
it can be not only icon, but something other, that's why i was expanded label-atom with <slot/>
not sure, that new `label-container` or something else will be better for just showing slot

Reply:
With `icon-label` you can combine two components without expansion of original component logic.

Reply:
slot needed not for only icon passing, 
but for `hover`, `tooltip` and `icon`.

Reply:
@dudiq as I see, @lamartire means that we need NEW component and don't need extent v-label?

Reply:
@ifiri yes, I mean that.